_targets.R

```diff
@@ -1,38 +1,83 @@
 library(targets)
-source("1_fetch/src/get_nwis_data.R")
+source("1_fetch/src/download_nwis_site_data.R")
+source("1_fetch/src/nwis_site_info.R")
 source("2_process/src/process_and_style.R")
 source("3_visualize/src/plot_timeseries.R")
 
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+# Global parameters
+parameterCd <- '00010'
+startDate <- "2014-05-01" 
+endDate <- "2015-05-01"
+
+
+# For each site, download data and write to csv
+#         Note: this feels inefficient and hard to scale up. How can we "loop" over
+#               multiple sites without making a target specified to each individually?
 p1_targets_list <- list(
   tar_target(
+    site_data_01427207, 
+    download_nwis_site_data(site_num = "01427207",
+                            parameterCd = parameterCd,
+                            startDate = startDate,
+                            endDate = endDate,
+                            out_dir = "1_fetch/out/"),
```

Comment:
As I discussed in an earlier comment, it is nice to show the full filepath at the top-level of a pipeline. Instead of passing in just the `out_dir` and hiding the filepath pattern to a user, move your paste to this line. 

```r
out_dir = paste0('1_fetch/out/, 'nwis_01427207', '_data.csv')
```

One other suggestion for constructing file paths - I find that using `sprintf()` instead of `paste()` to be extremely helpful because it keeps the path in one string rather than splitting it up. For example, the above could be written as

```r
out_dir = sprintf("1_fetch/out/nwis_%s_data.csv", "01427207"))
```

The site number is inserted into the `%s` placeholder.